3.7/init.sh

```diff
@@ -46,6 +46,7 @@ cd ${PROJECT_DIR}
 sudo -u nominatim nominatim import --osm-file $OSMFILE --threads $THREADS
 sudo -u nominatim nominatim admin --check-database
 sudo -u nominatim nominatim replication --init
+sudo -u nominatim nominatim refresh --functions
```

Comment:
You only need to call this when you change the MODULE_PATH. As nothing has changed here since the import, no need to refresh here.